conda/libmesh/build.sh

```diff
@@ -74,8 +74,8 @@ fi
 
 source $SRC_DIR/configure_libmesh.sh
 export INSTALL_BINARY="${SRC_DIR}/build-aux/install-sh -C"
-METHODS="opt oprof devel dbg" configure_libmesh --prefix=${PREFIX}/libmesh \
-                                                --with-vtk-lib=${BUILD_PREFIX}/libmesh-vtk/lib \
+export LIBMESH_DIR=${PREFIX}/libmesh
+METHODS="opt oprof devel dbg" configure_libmesh --with-vtk-lib=${BUILD_PREFIX}/libmesh-vtk/lib \
```

Comment:
```suggestion
LIBMESH_DIR=${PREFIX}/libmesh METHODS="opt oprof devel dbg" configure_libmesh --with-vtk-lib=${BUILD_PREFIX}/libmesh-vtk/lib \
```

I would say do this; try to export as little as possible with conda